src/test/fixtures/profiles/processed-profile.js

```diff
@@ -1096,14 +1106,12 @@ export function getMergedProfileFromTextSamples(
     profiles,
     profiles.map(() => profileState)
   );
+  const funcNamesDictPerThread = profile.threads.map(getFuncNamesDictForThread);
+  const funcNamesPerThread = funcNamesDictPerThread.map(Object.keys);
```

Comment:
Oops, I think this way of computing `funcNamesPerThread` is not guaranteed to have the right strings at the right indexes. It'll probably mostly work because the keys of JS objects are ordered, but it would probably break if the same function name was used by two different func indexes. I'll fix this.